### ISSUE TITLE
Media player should not be in a figure element

### ISSUE THREAD

**Issue by ryanmccombe** [bbc/simorgh#9625]
**Describe the bug**
Our media player currently always renders inside a `figure` element:

![Screen Shot 2021-11-02 at 15 11 10](https://user-images.githubusercontent.com/8601460/139874808-d5448f09-720d-4b24-a72f-5413325f1d81.png)

Example Asset: https://www.test.bbc.com/thai/23463609

We've been asked to change this as it degrades screen reader experience.

For scenarios where the media player does not have a caption (which is almost always) this wrapping figure should ideally be removed.  If removing is not straightforward, we can change it to a `div` instead.
